ci-operator/config/openshift/kubernetes-metrics-server/openshift-kubernetes-metrics-server-release-4.14.yaml

```
@@ -0,0 +1,69 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.20-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+build_root:
+  from_repository: true
+canonical_go_repository: https://github.com/kubernetes-sigs/metrics-server
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.20-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14
+  to: metrics-server
+promotion:
+  disabled: true
+  name: "4.14"
+  namespace: ocp
```

```suggestion
promotion:
  name: "4.14"
  namespace: ocp
```
This section should enabled the automatic branch creation (less the disabled line). Not sure if we need to merge this with the rehearse jobs failing before it can take effect. I'll find out.